src/test/java/org/opensearch/neuralsearch/query/aggregation/BaseAggregationsWithHybridQueryIT.java

```diff
@@ -0,0 +1,253 @@
+    protected static final String TEST_DOC_TEXT1 = "Hello world";
+    protected static final String TEST_DOC_TEXT2 = "Hi to this place";
+    protected static final String TEST_DOC_TEXT3 = "We would like to welcome everyone";
+    protected static final String TEST_DOC_TEXT4 = "Hello, I'm glad to you see you pal";
+    protected static final String TEST_DOC_TEXT5 = "People keep telling me orange but I still prefer pink";
+    protected static final String TEST_DOC_TEXT6 = "She traveled because it cost the same as therapy and was a lot more enjoyable";
+    protected static final String TEST_TEXT_FIELD_NAME_1 = "test-text-field-1";
+    protected static final String TEST_QUERY_TEXT2 = "blah";
+    protected static final String TEST_QUERY_TEXT3 = "hello";
+    protected static final String TEST_QUERY_TEXT4 = "cost";
+    protected static final String TEST_QUERY_TEXT5 = "welcome";
+    protected static final String TEST_NESTED_TYPE_FIELD_NAME_1 = "user";
+    protected static final String NESTED_FIELD_1 = "firstname";
+    protected static final String NESTED_FIELD_2 = "lastname";
+    protected static final String NESTED_FIELD_1_VALUE_1 = "john";
+    protected static final String NESTED_FIELD_2_VALUE_1 = "black";
+    protected static final String NESTED_FIELD_1_VALUE_2 = "frodo";
+    protected static final String NESTED_FIELD_2_VALUE_2 = "baggins";
+    protected static final String NESTED_FIELD_1_VALUE_3 = "mohammed";
+    protected static final String NESTED_FIELD_2_VALUE_3 = "ezab";
+    protected static final String NESTED_FIELD_1_VALUE_4 = "sun";
+    protected static final String NESTED_FIELD_2_VALUE_4 = "wukong";
+    protected static final String NESTED_FIELD_1_VALUE_5 = "vasilisa";
+    protected static final String NESTED_FIELD_2_VALUE_5 = "the wise";
+    protected static final String INTEGER_FIELD_1 = "doc_index";
+    protected static final int INTEGER_FIELD_1_VALUE = 1234;
+    protected static final int INTEGER_FIELD_2_VALUE = 2345;
+    protected static final int INTEGER_FIELD_3_VALUE = 3456;
+    protected static final int INTEGER_FIELD_4_VALUE = 4567;
+    protected static final String KEYWORD_FIELD_1 = "doc_keyword";
+    protected static final String KEYWORD_FIELD_1_VALUE = "workable";
+    protected static final String KEYWORD_FIELD_2_VALUE = "angry";
+    protected static final String KEYWORD_FIELD_3_VALUE = "likeable";
+    protected static final String KEYWORD_FIELD_4_VALUE = "entire";
+    protected static final String DATE_FIELD_1 = "doc_date";
+    protected static final String DATE_FIELD_1_VALUE = "01/03/1995";
+    private static final String DATE_FIELD_2_VALUE = "05/02/2015";
+    protected static final String DATE_FIELD_3_VALUE = "07/23/2007";
+    protected static final String DATE_FIELD_4_VALUE = "08/21/2012";
+    protected static final String INTEGER_FIELD_PRICE = "doc_price";
+    protected static final int INTEGER_FIELD_PRICE_1_VALUE = 130;
```

Comment:
can we not move this to method instead of defining this as variable?

Reply:
not sure I got your point Vijay. We need to declare those variables, if we split declaration and initialization it will increase length of the code without boost in readability.